src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1012,6 +1012,7 @@ private function addCacheSection(ArrayNodeDefinition $rootNode)
                                     ->ifTrue(function ($v) { return (isset($v['adapters']) || \is_array($v['adapter'] ?? null)) && isset($v['provider']); })
                                     ->thenInvalid('Pool cannot have a "provider" while "adapter" is set to a map')
                                 ->end()
+                                ->performNoDeepMerging()
```

Comment:
this should be moved two lines below, under "adapters"

Reply:
Ah ok, wasn't sure where exactly it has to be placed. fixed it.